lib/src/pluto_grid_configuration.dart

```diff
@@ -370,7 +378,7 @@ class PlutoGridStyleConfig {
   final Color menuBackgroundColor;
 
   /// Set the border color of [PlutoGrid].
-  final Color gridBorderColor;
+  final Color? gridBorderColor;
```

Comment:
The ? makes possible to put a null value and removes the necessity of a border around the pluto grid.